mta/mta_services.go

```diff
@@ -3,15 +3,15 @@ package mta
 import (
 	"crypto/sha1"
 	"fmt"
-	"gopkg.in/yaml.v3"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"gopkg.in/yaml.v3"
```

Comment:
Why did you move it to a separate import group?

Reply:
I didn't... when I save it moves by itself ... I'll put it back